### ISSUE TITLE
Section 2: Reword para?

### ISSUE THREAD

**Issue by elear** [intarchboard/draft-use-it-or-lose-it#71]
The draft currently contains the following paragraph:

> Interoperability with other implementations is usually highly valued, so
deploying mechanisms that trigger adverse reactions can be untenable.  Where
interoperability is a competitive advantage, this is true even if problems are
infrequent or only occur under relatively rare conditions.

This is a bit opaque.  Might I suggest the following reword:

> Protocols generally require the backward compatibility necessary for implementations to properly interoperate.  

But perhaps that doesn't cover the thrust you intend?

**Comment by martinthomson** [comment]
This might be made clearer.  The point it makes is that interoperability is valuable (but that is basically a non-statement).  The key is that the degree to which interoperability failures can be tolerated can be exceedingly small.  In the web context, something that affects one in a thousand times people can cause something to be completely non-viable.  People do thousands of things, so it could mean that an event at millions to one odds can totally block deployment of a new capability.

In other words, it can be surprising just how small an interoperability risk can be before it affects our ability to deploy something.
